README.md

```diff
@@ -1 +1,223 @@
-# music-library
+# Music-Library
```

Comment:
Nice readme, loads of details, nice and clear about setup and available routes. Might be wroth considering using a table to hold all the routes to make it easier to read without having to scroll down the page so much, eg

| Method | Route | Description | Schema |
|------|------|------|------|
| POST | /artist | Creates a new artist | {<br>"name": STRING,<br>    "genre": STRING<br>}|
| GET | /artist | Returns all artists | N/A |

Heres a helpful cheatsheet [guide to markdown](https://www.markdownguide.org/cheat-sheet/)

Also useful is [markdown editor](https://jbt.github.io/markdown-editor/) so you can play with what the file will look like without having to do multiple commits to the repo to see the results